web/src/Search.js

```diff
@@ -0,0 +1,37 @@
+import React, { Component } from 'react';
+import './App.css';
+
+export class Search extends Component {
+  constructor(props) {
+    super(props);
+    this.state = { value: '' };
+  }
+
+  handleChange = event => {
+    this.setState({ value: event.target.value });
+  };
+
+  handleOnClick = event => {
+    this.props.onSubmit(this.state.value);
+  };
+
+  render() {
+    return (
+      <body className="body">
```

Comment:
You cannot use `<body/>` here:

<img width="496" alt="screen shot 2017-10-17 at 22 43 06" src="https://user-images.githubusercontent.com/2963462/31663089-9566fecc-b38c-11e7-8880-80428116ea2d.png">

You can just change it to `<div/>`.

Reply:
Ok, I delete this.